scripts/irods/database_upgrade.py

```diff
@@ -69,6 +69,12 @@ def run_update(irods_config, cursor):
         database_connect.execute_sql_statement(cursor, "create index idx_data_main7 on R_DATA_MAIN (resc_id);")
         database_connect.execute_sql_statement(cursor, "create index idx_data_main8 on R_DATA_MAIN (data_is_dirty);")
 
```

Comment:
will need to bump the db version elsewhere and then check it here...
```
    elif new_schema_version == 7:
```

Reply:
Is the bumping of the db version supposed to be part of this PR? Not sure about that, but if-else has been added.

Reply:
yes, i think so - if we're bumping the logic to 7... we need to tell 4.2.8 the db should be at 7.    you could double tag a separate commit with this issue and the 4.2.8 release activities - so it's easy to figure out later.